Update documentation (#3156)
openvinotoolkit/open_model_zoo: demos/README.md

Please run the following command before the demos build (assuming that the binary package was installed to `<INSTALL_DIR>`):

```sh
<INSTALL_DIR>/extras/scripts/download_opencv.sh
source <INSTALL_DIR>/setupvars.sh
```

> > **NOTE:** If you are planning to use Python\* demos only, you can install OpenVINO Python\* package.

Reviewer: ```suggestion
> **NOTE:** If you plan to use Python\* demos only, you can install the OpenVINO Python\* package.
```

> ```sh
> pip install openvino
> ```

> You can also build demos manually using Inference Engine built from the [openvino](https://github.com/openvinotoolkit/openvino) repo. In this case please set `InferenceEngine_DIR` environment variable to a folder containing `InferenceEngineConfig.cmake` and `ngraph_DIR` to a folder containing `ngraphConfig.cmake` in a build folder. Please also set the `OpenCV_DIR` to point to the OpenCV package to use. The same OpenCV version should be used both for Inference Engine and demos build. Alternatively these values can be provided via command line while running `cmake`. See [CMake's search procedure](https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure).

Reviewer 2: There still references to Inference Engine and `InferenceEngine_DIR`

Author: updated references to Inference Engine, except `InferenceEngine_DIR` is an environment variable to `InferenceEngineConfig.cmake` file directory

Reviewer 2: @jkamelin I think InferenceEngineConfig.cmake still maintained for needs of applications built on old API. Suggest to advice customers to stick to new API instead.
@ilya-lavrenov seems the page https://github.com/openvinotoolkit/openvino/wiki/BuildingCode outdated, could you please advice on how we need to describe build OpenVINO ?

Reviewer 2: according to @ilya-lavrenov now it should be OpenVINO_DIR

Reviewer 3: right, instead of ngraph and IE configs we need to use `OpenVINOConfig.cmake`, in cmake cmd line we need to point `OpenVINO_DIR` location.
As a target to link application with - use `openvino::runtime` instead of ngraph and IE ones

Author: Ok, looks like we need to update our demos first, because this variables are used to build them.

### Get Ready for Running the Demo Applications on Windows*

Before running compiled binary files, make sure your application can find the Inference Engine and OpenCV libraries.

> Optionally, download the OpenCV community FFmpeg plugin using the downloader script in the OpenVINO package: `<INSTALL_DIR>\extras\opencv\ffmpeg-download.ps1`.

Reviewer 2: OpenCV is not optional but mandatory dependence for almost all OMZ demos, both for build and run

Author: This is about OpenCV FFmpeg plugin on windows. Instructions about OpenCV downloading is build section https://github.com/jkamelin/open_model_zoo/blob/jk/documentation/demos/README.md?plain=1#L93-L98. Before run we execute `setupvars`, assuming opencv is installed https://github.com/jkamelin/open_model_zoo/blob/jk/documentation/demos/README.md?plain=1#L328

Reviewer 2: ok, then
